os/kernel/binary_manager/binary_manager.h

```diff
@@ -145,6 +144,7 @@ extern binmgr_bininfo_t bin_table[BINARY_COUNT];
  *
  ****************************************************************************/
 void binary_manager_recovery(int pid);
+mqd_t binary_manager_get_mqfd(void);
```

Comment:
How about add `#ifdef CONFIG_BINMGR_RECOVERY` ?

Reply:
It has already dependency on CONFIG_BINMGR_RECOVERY.